src/components/Main.jsx

```diff
@@ -0,0 +1,64 @@
+import style from './Main.module.css';
+import person from '../../images/illustration-working.svg?react';
+import brand from '../../images/icon-brand-recognition.svg?react';
+import detailed from '../../images/icon-detailed-records.svg?react';
+import fully from '../../images/icon-fully-customizable.svg?react';
+
+function Main() {
+  return (
+    <div className={style.main}>
+      <div className={style.more_img}>
+        <section className={style.more}>
+          <h1 className={style.more_then}>More than just shorter links</h1>
+          <p className={style.p}>
+            Build your brand&apos;s and get detailed insights on how your links are performing
+          </p>
+          <button className={style.getStarted}>Get Started</button>
+        </section>
+        <img className={style.person} src={person} alt="person" />
+      </div>
+      <div className={style.shorted}>
```

Comment:
Why the whole page is only one component? Where are reusable components?

Reply:
I agree with  you,
I made a button 'get started' but elements with icons I think are not responsible  make  components be more complex than now

Reply:
When you develop something you always should think about future usage. The apps tend to change in time, new requirements, new features. If you learn only how to create something for "today" usage you will struggle tomorrow.